README.md

```diff
@@ -1,46 +1,77 @@
-# Getting Started with Create React App and Redux
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+# Book a class
 
-## Available Scripts
+>  This is the final group capstone project - front-end. The back-end is Book-a-class. Please, see the link below:
 
-In the project directory, you can run:
+## Link to the back-end repo
 
-### `npm start`
+[Book-a-class](https://github.com/Hombre2014/Book-a-class)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Project requirements
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+The project requirements can be viewed [here](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)
 
-### `npm test`
+## Built With
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```

Comment:
- Consider adding live demo link
- Maybe add some screenshots (not too many, enough, even if it's a link to screenshots) (especially if you don't have a demo link) How would person have an idea of what the project looks like before they clone it? This is important to do

Reply:
Thanks for your review. Sure, for the link we are experiencing some problems with deployment. We have tried three different back-end deployments and they failed. That is why we still do not have a live demo link. We will add some screenshots though.

Reply:
Can you send the errors your getting? I may be able to help

Reply:
Hi, @mcrd25 ! You are very kind! I really appreciate your enthusiasm. OK, here is the problem with deploying the back-end. Keep in mind that this is API endpoints back end and there are no views.
This is from heroku logs --tail:

`yuriy@ubuntu:~/Projects/Microverse/Capstone/Mubarak/Book-a-class$ heroku open
yuriy@ubuntu:~/Projects/Microverse/Capstone/Mubarak/Book-a-class$ heroku logs --tail
2022-05-20T19:59:34.522661+00:00 app[web.1]: from /app/config/environment.rb:5:in `<main>'
2022-05-20T19:59:34.522662+00:00 app[web.1]: from config.ru:3:in `require_relative'
2022-05-20T19:59:34.522663+00:00 app[web.1]: from config.ru:3:in `block in <main>'
2022-05-20T19:59:34.522664+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:116:in `eval'
2022-05-20T19:59:34.522671+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:116:in `new_from_string'
2022-05-20T19:59:34.522671+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:105:in `load_file'
2022-05-20T19:59:34.522672+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:66:in `parse_file'
2022-05-20T19:59:34.522679+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:349:in `build_app_and_options_from_config'
2022-05-20T19:59:34.522679+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:249:in `app'
2022-05-20T19:59:34.522681+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:422:in `wrapped_app'
2022-05-20T19:59:34.522687+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:312:in `block in start'
2022-05-20T19:59:34.522687+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:379:in `handle_profiling'
2022-05-20T19:59:34.522689+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:311:in `start'
2022-05-20T19:59:34.522695+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands/server/server_command.rb:38:in `start'
2022-05-20T19:59:34.522695+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands/server/server_command.rb:143:in `block in perform'
2022-05-20T19:59:34.522697+00:00 app[web.1]: from <internal:kernel>:90:in `tap'
2022-05-20T19:59:34.522704+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands/server/server_command.rb:134:in `perform'
2022-05-20T19:59:34.522704+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
2022-05-20T19:59:34.522705+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
2022-05-20T19:59:34.522707+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
2022-05-20T19:59:34.522713+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/command/base.rb:87:in `perform'
2022-05-20T19:59:34.522713+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/command.rb:48:in `invoke'
2022-05-20T19:59:34.522715+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands.rb:18:in `<main>'
2022-05-20T19:59:34.522722+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
2022-05-20T19:59:34.522722+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
2022-05-20T19:59:34.522724+00:00 app[web.1]: from bin/rails:4:in `<main>'
2022-05-20T19:59:34.649820+00:00 heroku[web.1]: Process exited with status 1
2022-05-20T19:59:34.780208+00:00 heroku[web.1]: State changed from starting to crashed
2022-05-20T19:59:34.783672+00:00 heroku[web.1]: State changed from crashed to starting
2022-05-20T19:59:38.442068+00:00 heroku[web.1]: Starting process with command `bin/rails server -p ${PORT:-5000} -e production`
2022-05-20T19:59:41.974334+00:00 app[web.1]: => Booting Puma
2022-05-20T19:59:41.974348+00:00 app[web.1]: => Rails 7.0.2.3 application starting in production
2022-05-20T19:59:41.974348+00:00 app[web.1]: => Run `bin/rails server --help` for more startup options
2022-05-20T19:59:42.684164+00:00 app[web.1]: Exiting
2022-05-20T19:59:42.684667+00:00 app[web.1]: /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/callbacks.rb:25:in `on_file_autoloaded': expected file /app/app/models/courses_reservation.rb to define constant CoursesReservation, but didn't (Zeitwerk::NameError)
2022-05-20T19:59:42.684669+00:00 app[web.1]: 
2022-05-20T19:59:42.684669+00:00 app[web.1]: raise Zeitwerk::NameError.new("expected file #{file} to define constant #{cpath}, but didn't", cref.last)
2022-05-20T19:59:42.684669+00:00 app[web.1]: ^^^^^
2022-05-20T19:59:42.684670+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:28:in `require'
2022-05-20T19:59:42.684670+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:95:in `const_get'
2022-05-20T19:59:42.684672+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:95:in `cget'
2022-05-20T19:59:42.684674+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:237:in `block (2 levels) in eager_load'
2022-05-20T19:59:42.684681+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:26:in `block in ls'
2022-05-20T19:59:42.684683+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:18:in `each_child'
2022-05-20T19:59:42.684684+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:18:in `ls'
2022-05-20T19:59:42.684691+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:232:in `block in eager_load'
2022-05-20T19:59:42.684693+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:217:in `synchronize'
2022-05-20T19:59:42.684694+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:217:in `eager_load'
2022-05-20T19:59:42.684701+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:317:in `each'
2022-05-20T19:59:42.684703+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:317:in `eager_load_all'
2022-05-20T19:59:42.684704+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/application/finisher.rb:78:in `block in <module:Finisher>'
2022-05-20T19:59:42.684727+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/initializable.rb:32:in `instance_exec'
2022-05-20T19:59:42.684736+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/initializable.rb:32:in `run'
2022-05-20T19:59:42.684738+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/initializable.rb:61:in `block in run_initializers'
2022-05-20T19:59:42.684739+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:228:in `block in tsort_each'
2022-05-20T19:59:42.684741+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
2022-05-20T19:59:42.684748+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:431:in `each_strongly_connected_component_from'
2022-05-20T19:59:42.684749+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:349:in `block in each_strongly_connected_component'
2022-05-20T19:59:42.684751+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:347:in `each'
2022-05-20T19:59:42.684759+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:347:in `call'
2022-05-20T19:59:42.684760+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:347:in `each_strongly_connected_component'
2022-05-20T19:59:42.684762+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:226:in `tsort_each'
2022-05-20T19:59:42.684769+00:00 app[web.1]: from /app/vendor/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:205:in `tsort_each'
2022-05-20T19:59:42.684770+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/initializable.rb:60:in `run_initializers'
2022-05-20T19:59:42.684771+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/application.rb:372:in `initialize!'
2022-05-20T19:59:42.684779+00:00 app[web.1]: from /app/config/environment.rb:5:in `<main>'
2022-05-20T19:59:42.684780+00:00 app[web.1]: from config.ru:3:in `require_relative'
2022-05-20T19:59:42.684782+00:00 app[web.1]: from config.ru:3:in `block in <main>'
2022-05-20T19:59:42.684789+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:116:in `eval'
2022-05-20T19:59:42.684790+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:116:in `new_from_string'
2022-05-20T19:59:42.684792+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:105:in `load_file'
2022-05-20T19:59:42.684799+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/builder.rb:66:in `parse_file'
2022-05-20T19:59:42.684800+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:349:in `build_app_and_options_from_config'
2022-05-20T19:59:42.684801+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:249:in `app'
2022-05-20T19:59:42.684809+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:422:in `wrapped_app'
2022-05-20T19:59:42.684811+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:312:in `block in start'
2022-05-20T19:59:42.684812+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:379:in `handle_profiling'
2022-05-20T19:59:42.684819+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/rack-2.2.3/lib/rack/server.rb:311:in `start'
2022-05-20T19:59:42.684820+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands/server/server_command.rb:38:in `start'
2022-05-20T19:59:42.684822+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands/server/server_command.rb:143:in `block in perform'
2022-05-20T19:59:42.684829+00:00 app[web.1]: from <internal:kernel>:90:in `tap'
2022-05-20T19:59:42.684830+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands/server/server_command.rb:134:in `perform'
2022-05-20T19:59:42.684832+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
2022-05-20T19:59:42.684839+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
2022-05-20T19:59:42.684840+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
2022-05-20T19:59:42.684842+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/command/base.rb:87:in `perform'
2022-05-20T19:59:42.684850+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/command.rb:48:in `invoke'
2022-05-20T19:59:42.684850+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/railties-7.0.2.3/lib/rails/commands.rb:18:in `<main>'
2022-05-20T19:59:42.684852+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
2022-05-20T19:59:42.684859+00:00 app[web.1]: from /app/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
2022-05-20T19:59:42.684861+00:00 app[web.1]: from bin/rails:4:in `<main>'
2022-05-20T19:59:42.813361+00:00 heroku[web.1]: Process exited with status 1
2022-05-20T19:59:42.860563+00:00 heroku[web.1]: State changed from starting to crashed
2022-05-20T20:02:03.378647+00:00 heroku[run.4458]: Awaiting client
2022-05-20T20:02:03.395551+00:00 heroku[run.4458]: Starting process with command `rails db:migrate`
2022-05-20T20:02:03.475693+00:00 heroku[run.4458]: State changed from starting to up
2022-05-20T20:02:11.298155+00:00 heroku[run.4458]: Process exited with status 0
2022-05-20T20:02:11.446532+00:00 heroku[run.4458]: State changed from up to complete
2022-05-20T20:02:33.308583+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=radiant-garden-34056.herokuapp.com request_id=6dbf6451-c5c5-47f5-ab96-607f2fc5dee3 fwd="88.77.19.60" dyno= connect= service= status=503 bytes= protocol=https
2022-05-20T20:02:34.702272+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=radiant-garden-34056.herokuapp.com request_id=3c340240-a77d-4abd-9e57-dc43998ab25a fwd="88.77.19.60" dyno= connect= service= status=503 bytes= protocol=https`

Reply:
Ok, I'm busy with work right now, but by EOD tomorrow I'd have a reply for you once I take a look at it.  Confident  :)